test/managers/course_repository_test.dart

```diff
@@ -1714,7 +1714,7 @@ void main() {
             programCode: '999',
             numberOfCredits: 3,
             title: 'Cours générique',
-            evaluation: evaluation);
+            review: evaluation);
```

Comment:
```suggestion
            review: review);
```
I think you just forgot this one (declared at line 1700) but the rest looks fine :)

Reply:
Thanks a lot Emma for the help on this 😄

Reply:
You're welcome, that kind of refactoring always ends up more painful than it should be 😅